config.py

```diff
@@ -65,6 +65,10 @@ class Config(object):
     # GitHub webhooks
     WEBHOOKS_SECRET = os.environ.get("WEBHOOKS_SECRET")
 
+    # Matomo
+    MATOMO_SERVER_URL = os.environ.get("MATOMO_SERVER_URL", "matomo-dev.acelab.ca")
```

Comment:
can you modify this to

```suggestion
    MATOMO_SERVER_URL = os.environ.get("MATOMO_SERVER_URL", <your Matomo URL>)
```

Reply:
The fallback value needs to be valid so I've removed it from here instead